solr/core/src/java/org/apache/solr/schema/SchemaManager.java

```diff
@@ -138,6 +138,7 @@ private List doOperations(List<CommandOperation> operations) throws InterruptedE
             //only for non cloud stuff
             managedIndexSchema.persistManagedSchema(false);
             core.setLatestSchema(managedIndexSchema);
+            core.getCoreContainer().reload(core.getName());
```

Comment:
I suppose this does look that easy.  Please run all tests to insure we've not broken something.

Reply:
I fave 2 test failing, which behave different when running with ant or in eclipse:

org.apache.solr.rest.schema.TestBulkSchemaAPI.testAnalyzerClass()
org.apache.solr.client.solrj.request.SchemaTest.addFieldTypeWithAnalyzerClassAccuracy()

Both are failing because luceneMatchVersion attribute of the analyzer configuration does not match, when running with ant it is always the latest version, in eclipse null is returned.

I am not sure what is the expected result when setting luceneMatchVersion on an analyzer.

Found additional NullPointer issues in some Schema test which are not related to the change but to the execution order of tests, fixed them to.

I pushed the changes, waiting for your feedback.

Reply:
After some code analysis it is 100% clear that the only analyzer property which get written to the schema is the class property look at method org.apache.solr.response.SchemaXmlWriter.writeAnalyzer(SimpleOrderedMap<Object>, String) at the lines 210 to 222, i will push the modification again